utils/appliance/endpoints/__init__.py

```diff
@@ -1,15 +0,0 @@
-class Endpoint(object):
```

Comment:
given the usage-pattern, ideally we never ever directly use "endpoints" as they only implmeent
so i dont see a need to type-check off-hand